modules/complete-onpremise-cluster/README.md

```diff
@@ -0,0 +1,16 @@
+```hct
+
+module "kubespray" {
```

Comment:
@devopsmher Does this need to be here? You seem to have the same in k8s-cluster module?

Reply:
all this is temporary, because it changes, we will create a readme at the end.